aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+ASP.NET Localization
+====================
+
+By `Tom Archer`_
+
+The ASP.NET 5 localization support provides middleware for establishing the correct culture and UI culture on the thread based on the request, and mechanisms for accessing localized content based on the current culture. In this article, you'll learn how to enable localization and establish the culture using both built-in culture providers. You'll also see how to implement your own custom culture provider.
+
+Enabling localization and establishing culture
+-----------------------------------------------
+
+To enable localization, add the request localization middleware to the request pipeline from the ``Startup`` class with a call to the ``IApplicationBuilder.UseRequestLocalization`` extension method. The ``IApplicationBuilder.UseRequestLocalization`` extension method has two overloaded versions: a parameter-less version that instantiates and uses a default ``RequestLocalizationOptions`` object and a version that allows you to specify a ``RequestLocalizationOptions`` object as a parameter.
```

Comment:
Before you add the middleware in startup you need to add the right package as a dependency in project.json.